addons/binding/org.openhab.binding.mihome/ESH-INF/thing/aqara_86sw2.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="86sw2">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Xiaomi Aqara 2 Channel Smart Light Control</label>
+		<description>Battery powered Aqara Switch with 2 channels</description>
+
+		<channels>
+			<channel id="ch1" typeId="sw1-trg" />
+			<channel id="ch2" typeId="sw2-trg" />
+			<channel id="dual_ch" typeId="dual_sw" />
+			<channel id="batteryLevel" typeId="system.battery-level" />
+			<channel id="lowBattery" typeId="system.low-battery" />
+		</channels>
+
+		<config-description>
+			<parameter name="itemId" type="text" readOnly="true">
+				<label>Item ID</label>
+				<description>An identifier for the item connected.</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="sw1-trg">
+		<kind>trigger</kind>
+		<label>Button 1 Event</label>
+		<event>
+			<options>
+				<option value="CLICK">click</option>
+				<option value="DOUBLE_CLICK">double click</option>
+			</options>
+		</event>
+	</channel-type>
+
+	<channel-type id="sw2-trg">
+		<kind>trigger</kind>
+		<label>Button 2 Event</label>
+		<event>
+			<options>
+				<option value="CLICK">click</option>
+				<option value="DOUBLE_CLICK">double click</option>
+			</options>
+		</event>
+	</channel-type>
+
+	<channel-type id="dual_sw">
+		<kind>trigger</kind>
+		<label>Both Button Event</label>
+		<event>
+			<options>
+				<option value="BOTH_CLICK">both click</option>
```

Comment:
No, it is superfluous and not recommended. It is not the name for your event, but it is an additional, unnecessary property of your event.